### ISSUE TITLE
Error: template: gitea/templates/gitea/gitea-ssh-svc.yaml

### ISSUE THREAD

**Issue by T0MASD** [jfelten/gitea-helm-chart#78]
Hi, I'm running:
```
/usr/local/bin/helm install gitea keyporttech/gitea --namespace=gitea
```
and getting an error:

Error: template: gitea/templates/gitea/gitea-ssh-svc.yaml:14:9: executing "gitea/templates/gitea/gitea-ssh-svc.yaml" at <(.Values.service.ssh.loadBalancerIP) and eq .Values.service.ssh.serviceType "LoadBalancer">: can't give argument to non-function .Values.service.ssh.loadBalancerIP

I'm using helm v3.3.4

**Comment by T0MASD** [comment]
I've tried cloning the repo and installing the chart manually:
```
$ git log -1 --oneline
5b506c3 (HEAD -> master, origin/master, origin/HEAD) Merge pull request #75 from jfelten/jfelten/fix_loadbalancer_config
$ helm install -n gitea --values ../values.yml gitea .
NAME: gitea
LAST DEPLOYED: Mon Oct  5 09:14:05 2020
NAMESPACE: gitea
STATUS: deployed
REVISION: 1
```
and it worked
it looks like load balancer fixes in 67c601d are not deployed on keyporttech/gitea (currently pointing to 5aeee51)

**Comment by T0MASD** [comment]
my bad, was not using the right repo: https://keyporttech.github.io/helm-charts/
